src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -55,14 +55,35 @@
  * a different <a href="https://checkstyle.org/property_types.html#scope">scope</a>.
  * </p>
  * <p>
- * Violates parameters and type parameters for which no param tags are present
- * can be suppressed by defining property {@code allowMissingParamTags}.
- * Violates methods which return non-void but for which no return tag is present
- * can be suppressed by defining property {@code allowMissingReturnTag}.
- * Violates exceptions which are declared to be thrown, but for which no throws
- * tag is present by activation of property {@code validateThrows}.
+ * Violates parameters and type parameters for which no param tags are present can
+ * be suppressed by defining property {@code allowMissingParamTags}.
  * </p>
  * <p>
+ * Violates methods which return non-void but for which no return tag is present can
+ * be suppressed by defining property {@code allowMissingReturnTag}.
+ * </p>
+ * <p>
+ * Violates exceptions which are declared to be thrown (by 'throws' in the method
+ * signature or by 'throw new' in the method body), but for which no throws tag is
+ * present by activation of property {@code validateThrows}.
+ * ATTENTION: Checkstyle does not have information about hierarchy of exception types
+ * so usage of base class is considered as separate exception type.
+ * As workaround you need to specify both types in javadoc (parent and exact type).
+ * Note that 'throw new' is not checked in the following places:
```

Comment:
Yes, I suppose it should. To make it more obvious that they are separate things I moved the Attention part below the list, so that it is clearly on its own paragraph.

New site preview:

![image](https://user-images.githubusercontent.com/53135010/80563473-dd515100-8a1d-11ea-94f8-3afbc1522b28.png)